src/ui/src/js/projects.js

```diff
@@ -23,6 +23,9 @@ window.$ = window.jQuery = jQuery;
 
 import "jstree";
 
+import dt from "datatables.net";
```

Comment:
```suggestion
import 'datatables.net'
import 'datatables.net-dt'
```

Seemed they were both needed.  I'll test tomorrow.